ginga/qtw/Widgets.py

```diff
@@ -1815,9 +1815,22 @@ def __init__(self, title=None, selectedfilter=None):
         self.widget = self.getSaveFileName(self, title, '', selectedfilter)
 
     def get_path(self):
-        if (self.widget and self.selectedfilter is not None and
-                not self.widget.endswith(self.selectedfilter[1:])):
-            self.widget += self.selectedfilter[1:]
+        if isinstance(self.widget, tuple):
+            w = self.widget[0]
+        else:
+            w = self.widget
+
+        if (w and self.selectedfilter is not None and
+                not w.endswith(self.selectedfilter[1:])):
+            w += self.selectedfilter[1:]
+
+        if isinstance(self.widget, tuple):
+            wlist = [x for x in self.widget]
```

Comment:
Haha... yes, I could... I'll test your changes next week. Thank you and have a great weekend!